src/test/skript/tests/regressions/5804-is-burning.sk

```diff
@@ -1,9 +1,14 @@
+# This test is to see if the `is burning` condition works properly.
```

Comment:
not sure this explanation is needed, they can just pull up the issue from the github right?

Reply:
They can, just saves a bit of time was my thinking behind it